### PR TITLE
hotfix: resolve docs and makefile run target

### PR DESCRIPTION
**Pull request by ramizpolic**
## Overview

Enables default execution to work on fresh pull when doing
```shell
make deps
make up
make run
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -7,6 +7,7 @@ IMG ?= ghcr.io/bank-vaults/vault-operator:dev
 TEST_K8S_VERSION ?= 1.27.1
 TEST_VAULT_VERSION ?= 1.13.3
 TEST_BANK_VAULTS_VERSION ?= 1.19.0
+TEST_BANK_VAULTS_IMAGE ?= ghcr.io/banzaicloud/bank-vaults:$(TEST_BANK_VAULTS_VERSION)
```
Comment (ramizpolic) [inline Makefile:10]: The latest version of bank vaults does not work, only a fixed 1.19.0 version

Comment (ramizpolic) [reply Makefile:10]: note: it requires additional security configuration which is not relevant for local operator development

Comment (akijakya) [reply Makefile:10]: bank-vaults should be fixed by [this](https://github.com/banzaicloud/bank-vaults/pull/1903) once its merged btw

**Review by akijakya — APPROVED** [review]
💯

File: Makefile (hunk 1 above)
Comment (ramizpolic) [reply Makefile:10]: awesome @akijakya, can we have a ticket for this?

Comment (ramizpolic) [reply Makefile:10]: thanks @akijakya, i have made a separate issue for this

**Pull request merged**